src/collectives/device/all_to_all.h

```diff
@@ -18,18 +18,24 @@ class ncclFunction<ncclFuncAllToAll, ALGO, PROTO, FUNC, T, UNROLL> {
       const int nChannels = args->coll.nChannels;
       struct ncclDevComm* comm = args->comm;
       struct ncclChannel* channel = comm->channels+blockIdx.x;
-      struct ncclRing* ring = &channel->ring;
+      struct scklGraph* graph = &channel->graph;
```

Comment:
Change the variable & field name from "graph" - this collides with the use of graph for NcclTopoGraph. Something like sgraph or sGraph (for scklGraph) might be better.

Reply:
fixed.